data/com/mojang/blaze3d/vertex/VertexConsumer.mapping

```diff
@@ -24,15 +29,15 @@ CLASS com/mojang/blaze3d/vertex/VertexConsumer
 		ARG 1 u
 		ARG 2 v
 	METHOD putBulkData (Lcom/mojang/blaze3d/vertex/PoseStack$Pose;Lnet/minecraft/client/renderer/block/model/BakedQuad;FFFII)V
-		ARG 1 matrixEntry
+		ARG 1 poseEntry
```

Comment:
Might want to document this as well for class names that are the same as field names as this does represent an entry in a `PoseStack`. Though, this doesn't have to be the case, it's assumed.